modules/ROOT/partials/configuration/type_ahead_urls.adoc

```diff
@@ -1,6 +1,7 @@
+[[typeahead_urls]]
 == `typeahead_urls`
 
-This option allows disable the type ahead url field feature.
+This option allows to disable the type ahead url field feature.
```

Comment:
This doesn't make sense? I think instead `disable` should have been changed to `disabling`:
```suggestion
This option allows disabling the type ahead url field feature.
```